src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -48,7 +48,7 @@ azure-mgmt-imagebuilder==0.4.0
 azure-mgmt-iotcentral==3.0.0
 azure-mgmt-iothub==0.12.0
 azure-mgmt-iothubprovisioningservices==0.2.0
-azure-mgmt-keyvault==7.0.0b3
+azure-mgmt-keyvault==8.0.0
```

Comment:
is there any change required in other command modules who depends on keyvault sdk?

Reply:
@yungezz The only breaking change is SKU family, I didn't find any usages of this out of the keyvault module. Related tests on other modules have been re-recorded.